docs/tutorials/getting-started-with-flask.rst

```diff
@@ -0,0 +1,114 @@
+Getting started with Rockcraft for Flask
+----------------------------------------
+
+In this tutorial you will learn how to create a Flask application, build a rock
+with the Flask application and run it.
+
+Prerequisites
+-------------
+- snap enabled system (https://snapcraft.io)
+- LXD installed (https://linuxcontainers.org/lxd/getting-started-cli/)
+- Docker installed (https://snapcraft.io/docker)
+- a text editor
+
+Create Flask application
+------------------------
+
+Create a new directory and write the following into a text editor and save it as
+``requirements.txt``:
+
+.. literalinclude:: code/getting-started-with-flask/requirements.txt
+
+Install ``python3-venv`` on your host and create a virtual environment:
+
+.. literalinclude:: code/getting-started-with-flask/task.yaml
+    :language: bash
+    :start-after: [docs:create-venv]
+    :end-before: [docs:create-venv-end]
+    :dedent: 2
+
+In the same directory, write the following into a text editor and save it as
+``app.py``:
+
+.. literalinclude:: code/getting-started-with-flask/app.py
+    :language: python
+
+Run and test the flask application to verify everything is working:
```

Comment:
(Context: I don't have experience with Flask applications) 
When I tried the tutorial, the `flask run &` seemed to work fine, but the `retry` command did not work. 
First, I wasn't sure whether I needed to CTRL+C from the `flask run &` command before running the other two commands. 
Then, when trying to run `retry -n 5 --wait 2 curl localhost:5000` I got the error `invalid option 'n'` and, when I removed that flag, `unrecognized option '--wait'`

Reply:
I removed the `retry`

Reply:
And added a note to explain what the `&` does